GOVERNANCE.md

```diff
@@ -181,24 +181,54 @@ yourself from the MAINTAINERS file.
 ## Removal of inactive maintainers
 
 Similar to the procedure for adding new maintainers, existing maintainers can
-be removed from the list if they do not show significant activity on the
-project. Periodically, the maintainers review the list of maintainers and their
-activity over the last year.
-
-If a maintainer has shown insufficient activity over this period, a neutral
-person will contact the maintainer to ask if they want to continue being
-a maintainer. If the maintainer decides to step down as a maintainer, they
-open a pull request to be removed from the MAINTAINERS file.
-
-If a committer is unable to continue participating in project level votes but
-wishes to remain a maintainer on the project, they may convert their role to
-a reviewer until they are able to perform committer duties again.
-
-If an inactive maintainer wants to remain in their current role or does not open
-their own pull request to change roles, another maintainer may open up a pull
-request to the MAINTAINERS file to change their role. Role changes and removals
-must be approved by 2/3 of the current committers. The voting and discussion
-period is seven days, after which a committer will verify the votes,
+be removed from the list if they do not show consistent activity on the
+project. Depending on the existing role, committer or reviewer, there are
+variations to the removal process detailed as follows.
+
+### Committer removal
+
+For committers, the maintainers periodically review committer project
+activity. The maintainers use the CNCF "devstats" community metrics tool for
+data on user activity statistics.
+
+If a committer has shown no activity over a six-month period, the committer
```

Comment:
```suggestion
If a committer has shown no activity over a six-month period, a process will begin to review the committer's status as an active maintainer. The committer
```

Reply:
Since you had 2 suggestions for the same sentence I took the latter one (below) as I believe the "process" part is detailed in the sentences following.